docs/_docs/installation.md

```diff
@@ -1,138 +1,164 @@
 
-## Install with RubyGems
+First, you need to [install Xcode from the App Store](https://itunes.apple.com/app/xcode/id497799835?mt=12) or [from Apple Developer website](https://developer.apple.com/xcode/). Then to install the command-line tools, open a terminal and run:
+
+```sh
+xcode-select --install
+```
 
-The best way to install Jekyll is via
-[RubyGems](https://rubygems.org/pages/download). At the terminal prompt,
-simply run the following command to install Jekyll:
+Check your Ruby version, and upgrade [RubyGems](https://rubygems.org/pages/download):
 
 ```sh
-gem install jekyll
+ruby -v
+2.3.3
+
+gem update --system
 ```
 
-All of Jekyll’s gem dependencies are automatically installed by the above
-command, so you won’t have to worry about them at all.
-
-<div class="note info">
-  <h5>Installing Xcode Command-Line Tools</h5>
-  <p>
-    If you run into issues installing Jekyll's dependencies which make use of
-    native extensions and are using macOS, you will need to install Xcode
-    and the Command-Line Tools it ships with. Download them in
-    <code>Preferences &#8594; Downloads &#8594; Components</code>.
-  </p>
-</div>
+Great, let's install Jekyll. We also need [bundler](https://bundler.io/) to help us handle [plugins](../plugins) and [themes](../themes):
 
-## Pre-releases
+```sh
+gem install bundler jekyll
+```
 
-In order to install a pre-release, make sure you have all the requirements
-installed properly and run:
+You're ready to go, either by installing our default jekyll blog theme with `jekyll new jekyll-website` or by starting from scratch:
 
 ```sh
-gem install jekyll --pre
+mkdir jekyll-website
+cd jekyll-website
+
+# Create a Gemfile
+bundle init
+
+# Add Jekyll
+bundle add jekyll
+
+# Install gems
+bundle install
 ```
 
-This will install the latest pre-release. If you want a particular pre-release,
-use the `-v` switch to indicate the version you'd like to install:
+From there you can either use a [theme](../themes/) or create your own layouts.
+
+### Install via Homebrew {#homebrew}
+
+You can install the latest version of Ruby via [Homebrew](https://brew.sh) a handy package manager for macOS.
 
 ```sh
-gem install jekyll -v '2.0.0.alpha.1'
+/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+brew install ruby
 ```
 
-If you'd like to install a development version of Jekyll, the process is a bit
-more involved. This gives you the advantage of having the latest and greatest,
-but may be unstable.
+### Install Rbenv
+
+Developers often use [rbenv](https://github.com/rbenv/rbenv) to manage multiple Ruby versions.
 
 ```sh
-git clone git://github.com/jekyll/jekyll.git
-cd jekyll
-script/bootstrap
-bundle exec rake build
-ls pkg/*.gem | head -n 1 | xargs gem install -l
+# Install rbenv and ruby-build
+brew install rbenv
```

Comment:
does the homebrew installation of rbenv include ruby-build?

Reply:
Just tried it...

`brew info` lists it as "recommended" for `rbenv` and will download the latest tarball during `rbenv` installation process.

On macOS 10.13.

Reply:
ok cool!